src/config.rs

```diff
@@ -1,10 +1,13 @@
+use crate::bot::UserId;
 use anyhow::{Context, Result};
 use serde::Deserialize;
+use std::collections::HashSet;
 
 #[derive(Deserialize, Clone, Debug)]
 pub struct Config {
     pub telegram: Telegram,
     pub data_storages: Data,
+    pub superusers: HashSet<UserId>,
```

Comment:
Make a new config sub-object, name it `access` or smth and pass that instead of directly referring to the `HashSet<UserId>`